framework/test/jest/unit/specs/modules/chain/transport/transport.spec.js

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { TransferTransaction } = require('@liskhq/lisk-transactions');
+const {
+	Transport,
+} = require('../../../../../../../src/modules/chain/transport');
+const jobsQueue = require('../../../../../../../src/modules/chain/utils/jobs_queue');
+
+describe('Transport', () => {
+	const defaultBroadcastInterval = 5000;
+	const defaultReleaseLimit = 25;
+
+	let transport;
+	let transactionPoolStub;
+	let synchronizerStub;
+	let blocksStub;
+	let loggerStub;
+	let processorStub;
```

Comment:
IMO and following the recent testing guidelines we agreed on, we shouldn't mock processor or blocks module, only the public interfaces that deal with I/O

Reply:
I think in the guidelines, it states that we should mock the external modules, in this case, other blocks and processors are external